sda-commons-dependency-check/build.gradle

```diff
@@ -27,7 +27,7 @@ dependencies {
   compile project(':sda-commons-server-s3')
   compile project(':sda-commons-server-s3-testing')
   compile project(':sda-commons-server-security')
-  compile project(':sda-commons-server-starter')
+  compile project(':sda-commons-server-starter-swagger')
```

Comment:
Please also add the new module to this file. This file is used to detect dependency conflicts

Reply:
done